index.html

```diff
@@ -186,30 +186,31 @@ <h3>
           a {{PointerLockOptions}} and a <dfn>cursor position</dfn> which is a
           pair of numbers representing the location of the system mouse cursor
           when the Pointer Lock State was entered (the same location that is
-          reported in `screenX`, `screenY`). The [=pointer-lock target=] must
-          receive all relevant user generated {{MouseEvent}} events: namely,
+          reported in `screenX`, `screenY`). The [=pointer-lock target=]
+          receives all relevant user generated {{MouseEvent}} events: namely,
           all user-generated `mousemove`, `mousedown`, `mouseup`, `click`,
           `dblclick`, `auxclick`, and `wheel` [[ui-events]]. No other elements
-          may receive these events while in [=pointer lock state=]. There will
-          be no dispatching of events that require the concept of a mouse
-          cursor: namely, `mouseenter`, `mouseleave`, `mouseover`, `mouseout`,
-          `drag`, and `drop`.
+          receive these events while in [=pointer lock state=]. There will be
+          no dispatching of events that require the concept of a mouse cursor:
+          namely, `mouseenter`, `mouseleave`, `mouseover`, `mouseout`, `drag`,
+          and `drop`.
         </p>
         <p>
           While in the [=pointer lock state=] if the [=pointer-lock options=]'
           {{PointerLockOptions/unadjustedMovement}} member is `true` the event
-          coordinates must not be affected by the underlying platform behaviors
-          such as mouse acceleration. In other words, UA must use the APIs
-          provided by the underlying platform to guarantee getting the raw
-          events. If the {{PointerLockOptions}}'
-          {{PointerLockOptions/unadjustedMovement}} member is `false` UA must
-          rely on the default behavior of the underlying platform regarding the
-          mouse acceleration.
+          coordinates shall not be affected by the underlying platform
```

Comment:
Nit: we tend to use "must" instead of "shall" at the w3c:

```suggestion
          coordinates must not be affected by the underlying platform
```

Reply:
Refer to the comment in https://github.com/w3c/pointerlock/pull/49#discussion_r1372377390, I use "will" to replace "shall" hope this is acceptable too.

Reply:
yep, that works.